### ISSUE TITLE
Update misc typos on home page

### ISSUE THREAD

**Issue by theecrit** [openoakland/openoakland.org#118]
## Description

Some minor home page improvements:

- [ ] Add http://oakca.us/hack-night on the Home page, right before the "Join us in Slack" it just says "You'll find us every Tuesday night on Zoom at ."

- [ ] Lowercase “brigade” in “As part of the Code for America **Brigade** network...” (its technically not a proper noun).

- [ ] Restyle “OpenOakland bridges technology and community for a thriving and equitable Oakland.” so it’s not a header style (using span or css class to style instead). Headers should be in sequential order on the screen, so skipping to an h5 or whatever it currently is breaks a11y best practices.

#### If you are working on this issue, consider the following ongoing issues:
- [ ] "Purify" Jekyll source files. Make the file you're working on all markdown, or all HTML, and not both.
